modules/specialized-guides/pages/salt/salt-connectivity.adoc

```diff
@@ -0,0 +1,156 @@
+[[salt-connectivity]]
+= Salt Connectivity
+
+Depending on the size and spreadness of the environment where {productname} is used some connectivity issues are possible.
+There are no common recommendations for all the possible use cases as the environments could be very different expecially if public clouds instances are involved.
+
+
+
+[[minions-connectivity]]
+== Minions Connectivity
+
+In case if clients are losing the connection to the {productname} Server (or {productname} Proxy if involved),
+they are unreachable from the {productname} Server web UI or using the CLI.
+To ensure if it's the case, it's better to check if the client is not reachable from the {productname} Server with [command]``salt MINION_ID test.ping``,
+while [command]``venv-salt-call test.ping`` or ([command]``salt-call test.ping`` in case if non-bundle salt is used on the client)
+is working fine on the client side, it is recommended to set `tcp_keepalive` parameters.
+
+The parameters with the values from the example below could be used as a starting point to look for the better combination, which could work better for certain environment.
```

Comment:
```suggestion
The parameters with the values from the example below could be used as a starting point to look for the better combination, which could improve performance for some environments.
```

Reply:
It's not about the performance, but mostly about improving the connection to prevent the cases of connection loss with no restoring.

Reply:
Ok, I see.  Maybe, "could stabilize connection"  or "could stabilize server-client connection" or "could make connection stable".  Some with "avoid interruption"?  If you think all this is misleading, we can keep your initial wording "work better".  (I wanted to get rid of the two "betters" in one long sentence.)

Reply:
I now think it is better to use your words.